### PR TITLE
add open graph meta tags and image for storybook

### PR DESCRIPTION
**Pull request by JoCa96**
Relates to #346 

Added relevant [open graph](https://ogp.me/) meta tags to enable rich website previews for the Storybook page.

### CODE REVIEW

File: packages/sit-onyx/.storybook/public/og-storybook-logo.webp
Comment (larsrickert) [inline packages/sit-onyx/.storybook/public/og-storybook-logo.webp:1]: The Storybook logo looks a little low resolution, do they offer it as SVG so we can use a higher resolution in the webp?

**Review by larsrickert — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/.storybook/public/og-storybook-logo.webp
Comment (JoCa96) [reply packages/sit-onyx/.storybook/public/og-storybook-logo.webp:1]: I improved the resolution

File: apps/docs/src/.vitepress/config.ts [hunk 1]
```diff
@@ -35,6 +35,13 @@ export default defineConfig({
     ],
     ["meta", { property: "og:site_name", content: "onyx" }],
     ["meta", { property: "og:image", content: "https://onyx.schwarz/images/og-logo.webp" }],
+    ["meta", { property: "og:image:type", content: "image/webp" }],
+    ["meta", { property: "og:image:width", content: "1200" }],
+    ["meta", { property: "og:image:height", content: "630" }],
+    ["meta", { property: "og:image", content: "https://onyx.schwarz/images/og-logo.jpg" }],
+    ["meta", { property: "og:image:type", content: "image/jpeg" }],
+    ["meta", { property: "og:image:width", content: "1200" }],
+    ["meta", { property: "og:image:height", content: "630" }],
```
Comment (larsrickert) [inline apps/docs/src/.vitepress/config.ts:44]: Don't they override each other because they have the same property name?

Comment (larsrickert) [reply apps/docs/src/.vitepress/config.ts:44]: Maybe its enough to just have the jpg version and remove webp?

**Review by larsrickert — CHANGES_REQUESTED** [review]

File: apps/docs/src/.vitepress/config.ts (hunk 1 above)
Comment (JoCa96) [reply apps/docs/src/.vitepress/config.ts:44]: According to https://ogp.me/#array you can have a property multiple times.

**Review by larsrickert — APPROVED** [review]

**Pull request merged**